<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMenu</class>
 <widget class="QMainWindow" name="WMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbResoluEq">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolucion de Ecuaciones</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRungeEdo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Metodo de Runge-Kutta EDO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRungeSis">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Metodo de Runge-Kutta Sistemas de Ecuaciones</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSimpson">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Integracion por Metodo de Simpson</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLineal">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Diferenciacion por Metodo Lineal de Diferencias Finitas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbIndice">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Indice de Algoritmos sin Interfaz Grafica</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbIndice</sender>
   <signal>clicked()</signal>
   <receiver>pbIndice</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
